calico_kubernetes/calico_kubernetes.py

```diff
@@ -12,13 +12,21 @@
 import logging
 from netaddr import IPAddress, AddrFormatError
 from logutils import configure_logger
+import pycalico
 from pycalico import netns
 from pycalico.datastore import IF_PREFIX, DatastoreClient
 from pycalico.util import generate_cali_interface_name, get_host_ips
 from pycalico.ipam import IPAMClient
-from pycalico.datastore_errors import PoolNotFound
+from pycalico.block import AlreadyAssignedError
 
 logger = logging.getLogger(__name__)
+pycalico_logger = logging.getLogger(pycalico.__name__)
+log_levels = {"DEBUG": logging.DEBUG,
```

Comment:
As @Symmetric said in his comment, you don't actually need the mapping I suggested, since setLevel accepts a string.

You can simply do:
```
logging_level = os.environ.get("LOG_LEVEL", "INFO")
```